client/src/posts/PostPage.jsx

```diff
@@ -1,12 +1,20 @@
 import React, { useEffect, useState } from 'react';
 import Post from '../components/Post';
-import { useParams, Link } from 'react-router-dom';
+import { useParams, Link, useHistory } from 'react-router-dom';
 
-const PostPage = ({blogApi}) => {
+const PostPage = ({blogApi, loggedInStatus, user_id}) => {
```

Comment:
Perhaps stick with a consistent naming approach so `userId` instead of `user_id`.